README.md

```diff
@@ -60,4 +82,6 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
+
```

Comment:
Appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) in your Acknowledgments section, as required by the [Creative Commons License](https://creativecommons.org/licenses/) ✔